package.json

```diff
@@ -43,6 +43,7 @@
     "react": "*"
   },
   "dependencies": {
+    "@cityofzion/neon-js": "^4.7.1",
```

Comment:
that's the most annoying part. to be estimated if it's good to dep too much on libs. if minimal integration would be possible without the lib, otherwise we can still use it.

recently @juan-cortes had bad surprise with Bignumber.js version mismatch so we'll be careful about these kind of dep (which bring many dep).

need to see if stuff can be inlined. I already did a bunch of inlining in https://github.com/LedgerHQ/ledger-live-common/blob/master/src/families/neo/hw-app-neo/crypto.js

Reply:
Yeah I've seen it ! I went with the pretty straightforward/fast implementation using existing libraries, we can think later about getting rid of those or when we support it from libcore side !